pnpm-lock.yaml

```diff
@@ -566,7 +566,7 @@ importers:
         specifier: ^5.3.1
         version: 5.3.1
       astro-sst:
-        specifier: 2.42.0
+        specifier: 2.43.0
```

Comment:
Not related to this PR but I just ran `pnpm install` and this came through so I figure it should be committed